WtWSMS/common/landed_titles/BP_e_britannia.txt

```diff
@@ -1746,34 +2240,32 @@ e_britannia = {
 				#North Germanic Cultures
 				name_list_jute = cn_beornice
 			}
-		
 			ai_primary_priority = {
 				if = {
 					limit = { culture = culture:anglo_saxon }
 					add = @better_than_the_alternatives_score
 				}
 			}
-			
 			c_northumberland = {
 				color = { 245 0 0 }
 				cultural_names = {
 					#Roman Cultures
-					name_list_roman = cn_segedenum
-					name_list_romano_gallic = cn_segedenum
-					name_list_romano_hispanic = cn_segedenum
-					name_list_romano_british = cn_segedenum
-					name_list_romano_african = cn_segedenum
-					name_list_romano_raetian = cn_segedenum
-					name_list_romano_thracian = cn_segedenum
-					name_list_romano_illyrian = cn_segedenum
-					name_list_romano_pannonian = cn_segedenum
-					name_list_dalmatian = cn_segedenum
-					name_list_romano_noric = cn_segedenum
-					name_list_romano_gothic = cn_segedenum
-					name_list_romano_dacian = cn_segedenum
-					name_list_romano_frank = cn_segedenum
-					name_list_romano_aramean = cn_segedenum
-					name_list_romano_vandalic = cn_segedenum
+					name_list_roman = cn_segodunum
+					name_list_romano_gallic = cn_segodunum
+					name_list_romano_hispanic = cn_segodunum
+					name_list_romano_british = cn_segodunum
+					name_list_romano_african = cn_segodunum
+					name_list_romano_raetian = cn_segodunum
+					name_list_romano_thracian = cn_segodunum
+					name_list_romano_illyrian = cn_segodunum
+					name_list_romano_pannonian = cn_segodunum
+					name_list_dalmatian = cn_segodunum
+					name_list_romano_noric = cn_segodunum
+					name_list_romano_gothic = cn_segodunum
+					name_list_romano_dacian = cn_segodunum
+					name_list_romano_frank = cn_segodunum
+					name_list_romano_aramean = cn_segodunum
+					name_list_romano_vandalic = cn_segodunum
```

Comment:
Why the change? https://en.wikipedia.org/wiki/Segedunum

Reply:
> Segedu

While the fort is primarily known as Segedunum, I think that this name is in fact corrupt. I used Segodunum as a purified version.

Reply:
> > Segedu
> 
> While the fort is primarily known as Segedunum, this name is in fact corrupt. I used Segodunum as a purified version (c.f. city of that name in Gaul).